### PR TITLE
Add docs

### PR DESCRIPTION
**Pull request by MattIPv4**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
+
+Copyright (c) 2011-2020 cdnjs team
```
Comment (xtuc) [inline LICENSE:3]: I think it's usually cdnjs contributors

Comment (MattIPv4) [reply LICENSE:3]: I think contributors might imply that anyone who makes a PR holds copyright. I can just remove `team` to make it simpler.

(This is what we have in cdnjs/cdnjs currently)

File: .github/ISSUE_TEMPLATE/library-request.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+title: "Add: xxx"
+labels: "🏷 Library Request"
+---
+
+<!-- Please fill out the information in this issue template so that we can
+efficiently process your request for a new library to be added to cdnjs -->
+
+<!-- To add a library to cdnjs, it must have a way for an auto-update mechanism
+to be used. This can either be via a Git repository that has versions tagged,
+with the required files included in the repository at the tag. Or, this can be
```
Comment (JakeMakesStuff) [inline .github/ISSUE_TEMPLATE/library-request.md:13]: I don't know about this `Or, ` here, it feels like this should be `Alternatively, ` personally.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+To help with ensuring all library JSON files added to cdnjs are of a standard format and contain a valid auto-update configuration as well as other information about the library, we have an automated CI that will review your JSON file in the PR.
+
+If the CI finds any issues with your JSON file it will mark itself as failed and add an annotation to the JSON file in the pull request 'Files' tab with information about the issues it found.
+
+Please edit your JSON file either in the GitHub browser interface or via your local clone to resolve issues highlighted.
+If you are unsure, please ask a cdnjs maintainer for help and we'll happily work with you to get your PR ready to merge.
+
+### Reviewing and merging
+
+As soon as your PR is passing the CI and a maintainer has time, they will manually review your pull request to ensure no issues have slipped past the automated CI.
```
Comment (JakeMakesStuff) [inline CONTRIBUTING.md:272]: Possibly a comma here? Not 100% tbh.

**Comment by Shaynomaino** [comment]
CONTRIBUTING.md #455

**Pull request merged**